dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -2599,22 +2636,56 @@ public MethodInvocationNode visitMethodInvocation(MethodInvocationTree tree, Voi
                     new MethodInvocationNode(tree, target, arguments, getCurrentPath());
 
             Set<TypeMirror> thrownSet = new HashSet<>();
+
+            Element methodElement = TreeUtils.elementFromTree(tree);
+            // detect the existence of ThrowsException annotation
+            AnnotationMirror throwAnno =
+                    annotationProvider.getDeclAnnotation(methodElement, ThrowsException.class);
+            if (throwAnno != null) {
+                TypeMirror thrown;
+                // get type of thrown exception
+                String cls =
+                        AnnotationUtils.getElementValueClassName(throwAnno, "value", false)
+                                .toString();
+                if (cls == null) { // thrown type is Throwable by default
+                    thrown = elements.getTypeElement("java.lang.Throwable").asType();
```

Comment:
The default is already decleared in the annotation as `Throwable`, is this line accessible in any way?